compiler/tests/success/x86-64/typealias_amd64.jazz

```diff
@@ -0,0 +1,30 @@
+param int bloc_size = 4046;
+type arch_size = u64;
+
+
+export fn memncopy 
+(
+    reg ptr arch_size[bloc_size] target, 
+    reg ptr arch_size[bloc_size] source,
+    reg arch_size size
+) 
+->  
+    reg ptr arch_size[bloc_size],
+    reg ptr arch_size[bloc_size] 
+{
+    reg arch_size i;
+    i=0;
+
+    while (i< size) {
+        target[i] = source[i];
+        i+=1;
+    }
+
+    return (target,source);
+}
+
+inline fn f(reg u64 p) {
+    stack u64 s;
+    s = 1;
+    [p] = s;
+}
```

Comment:
What is this?

Reply:
Something I didn't have before (probably a typo)

Reply:
it doesn't test the feature so I will remove it